<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1377</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>421</width>
      <height>391</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="goNorthButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>530</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>North</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goSouthButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>630</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>South</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goWestButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>580</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string> West</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goEastButton">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>580</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>East</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="playerItemsTreeWidget">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>100</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
    <property name="indentation">
     <number>8</number>
    </property>
    <attribute name="headerDefaultSectionSize">
     <number>95</number>
    </attribute>
    <column>
     <property name="text">
      <string>Player Items</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Weight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Attack Power</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>HP Restore</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="goUpButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goDownButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>620</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string> Down</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="playerHPLcdNumber">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 0);</string>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QTreeWidget" name="itemsInRoomTreeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="indentation">
     <number>8</number>
    </property>
    <attribute name="headerDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="headerMinimumSectionSize">
     <number>15</number>
    </attribute>
    <column>
     <property name="text">
      <string>Items In Room</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Weight</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="charaInRoomTreeWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>410</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="indentation">
     <number>8</number>
    </property>
    <attribute name="headerCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>115</number>
    </attribute>
    <attribute name="headerMinimumSectionSize">
     <number>20</number>
    </attribute>
    <column>
     <property name="text">
      <string>Characters In Room</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">Health</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Weapon</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="attackButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>570</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack</string>
    </property>
   </widget>
   <widget class="QLabel" name="hpLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>HP:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="talkButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>570</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Talk</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equipButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>(Un)Equip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eatButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Use</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QLabel" name="weaponLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Weapon:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="eqWeaponLabel">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>No Weapon Equiped</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>262</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="attackLcdNumber">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="attackLabel">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Attack:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mapButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>580</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Volume:</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="renderHints">
     <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mute</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="countdownLcdNumber">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>440</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS UI Gothic</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 8pt &quot;MS UI Gothic&quot;;</string>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QTreeWidget" name="interactiveObjectsTreeWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="indentation">
     <number>8</number>
    </property>
    <column>
     <property name="text">
      <string>Interactive Objects</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="interactButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>570</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Interact</string>
    </property>
   </widget>
   <widget class="QLabel" name="countdownLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>400</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Countdown:</string>
    </property>
   </widget>
   <widget class="GLWidget2" name="widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>310</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
    <widget class="GLWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>191</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1377</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLWidget2</class>
   <extends>QWidget</extends>
   <header>glwidget2.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
